### PR TITLE
docs: use subscription

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

**Comment by atian25** [comment]
旧的方式要不要先保留着？ 就像 Controller 那样

**Comment by popomore** [comment]
文档开头有写

**Comment by popomore** [comment]
继续看看

File: README.md [hunk 1]
```diff
@@ -73,17 +96,21 @@ To create a task, it is as simple as write a generator / async function. For exa
 
 ```js
 // A simple logger example
-exports.task = function* (ctx) {
-  ctx.logger.info('Info about your task');
-};
+class LoggerExample extends Subscription {
```
Comment (atian25) [inline README.md]: 上面 `Task is a generator/async function` 这整段的描述感觉要全改改，其他 +1

**Comment by popomore** [comment]
@atian25 改了

**Comment by popomore** [comment]
+ egg-schedule@2.6.0

**Pull request merged**